src/yunohost/app.py

```diff
@@ -140,7 +144,7 @@ def app_fetchlist(url=None, name=None):
             raise MoulinetteError(errno.EBADR,
                                   m18n.n('appslist_retrieve_error', error=str(e)))
 
-        if (applist_request.status_code != 200):
+        if applist_request.status_code != 200:
```

Comment:
Shouldn't we continue if there are more than one app list to get? To avoid blocking all next lists if one fails.